.drone.yml

```diff
@@ -1,3 +1,226 @@
+      NPM_EMAIL:
+        from_secret: NPM_EMAIL
+      REALM_API_URL:
+        from_secret: REALM_API_URL
+      REALM_SEARCH_URL:
+        from_secret: REALM_SEARCH_URL_STAGING
+      STRAPI_URL:
+        from_secret: STRAPI_URL_STAGING
+      STRAPI_API_TOKEN:
+        from_secret: STRAPI_API_TOKEN
+      CS_BRANCH:
+        from_secret: CS_BRANCH
+      CS_DELIVERY_TOKEN:
+        from_secret: CS_DELIVERY_TOKEN
+      CS_GRAPHQL_URL:
+        from_secret: CS_GRAPHQL_URL
+      SENTRY_AUTH_TOKEN:
+        from_secret: SENTRY_AUTH_TOKEN
+      SENTRY_DSN:
+        from_secret: SENTRY_DSN
+      NEXT_IMAGE_HOSTS:
+        from_secret: NEXT_IMAGE_HOSTS
+      MONGODB_TV_API_KEY:
+        from_secret: MONGODB_TV_API_KEY
+      DEBIAN_FRONTEND: noninteractive
+  - name: unit-tests
+    image: ubuntu:focal
+    commands:
+    - apt-get -y update
+    - apt-get -y install --no-install-recommends ca-certificates curl
+    - curl -sL https://deb.nodesource.com/setup_16.x | bash
+    - apt-get -y install --no-install-recommends nodejs
+    - npx jest
+    resources:
+      limits:
+        memory: 18Gi
+    environment:
+      ACCOUNT_PORTAL_URL: https://account-qa.mongodb.com
+      APP_ENV: staging
+      HOST_URL: mongodbcom-cdn.staging.corp.mongodb.com
+      NPM_AUTH:
+        from_secret: NPM_AUTH
+      NPM_EMAIL:
+        from_secret: NPM_EMAIL
+      REALM_API_URL:
+        from_secret: REALM_API_URL
+      REALM_SEARCH_URL:
+        from_secret: REALM_SEARCH_URL_STAGING
+      STRAPI_URL:
+        from_secret: STRAPI_URL_STAGING
+      STRAPI_API_TOKEN:
+        from_secret: STRAPI_API_TOKEN
+      CS_BRANCH:
+        from_secret: CS_BRANCH
+      CS_DELIVERY_TOKEN:
+        from_secret: CS_DELIVERY_TOKEN
+      CS_GRAPHQL_URL:
+        from_secret: CS_GRAPHQL_URL
+      SENTRY_AUTH_TOKEN:
+        from_secret: SENTRY_AUTH_TOKEN
+      SENTRY_DSN:
+        from_secret: SENTRY_DSN
+      NEXT_IMAGE_HOSTS:
+        from_secret: NEXT_IMAGE_HOSTS
+      MONGODB_TV_API_KEY:
+        from_secret: MONGODB_TV_API_KEY
+      DEBIAN_FRONTEND: noninteractive
+  - name: lighthouse
+    image: ubuntu:focal
+    commands:
+    - apt-get -y update
+    - apt-get -y install --no-install-recommends ca-certificates curl libappindicator1 fonts-liberation libnss3-dev wget gnupg2
+    - echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" | tee -a /etc/apt/sources.list.d/google.list
+    - wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | apt-key add -
+    - curl -sL https://deb.nodesource.com/setup_16.x | bash -
+    - apt-get update
+    - apt-get install -y nodejs google-chrome-stable
+    - npm install --global yarn
+    - yarn install --frozen-lockfile
+    - yarn build
+    - export CHROME_PATH=$(which google-chrome-stable)
+    - export LHCI_BUILD_CONTEXT__CURRENT_BRANCH={$DRONE_BRANCH}
+    - yarn global add @lhci/cli@0.12.x
+    - lhci autorun
+    resources:
+      limits:
+        memory: 18Gi
```

Comment:
Will this memory limit connect to devrel name space ?

Reply:
if yes, try to give optimal value. Kanopy team is going to send resource utilization converted into $$$ equivalent and send to org owners.

Reply:
@harikakotipalli Ahh ok I can do that. Do you have any estimate of what would be a good limit here?

Reply:
here are the efficiency dashboards for [staging](https://grafana.staging.corp.mongodb.com/d/43f3f1511e5d14ecd08f95e773357949/namespace-resource-efficiency?orgId=1&var-datasource=default&var-namespace=devrel&var-num_pods=10&var-workload=All&var-instance=All&var-component=All&var-version=All) and [prod](https://grafana.corp.mongodb.com/d/43f3f1511e5d14ecd08f95e773357949/namespace-resource-efficiency?orgId=1&var-datasource=default&var-namespace=devrel&var-num_pods=40&var-workload=All&var-instance=All&var-component=All&var-version=All)

Reply:
Actually since these are being run in drone containers I don't think they will be in the devrel namespace. I will adjust though, 18G seems like a lot.